config/routes.rb

```diff
@@ -0,0 +1,25 @@
+Rails.application.routes.draw do
+  get 'attendances/create'
+  get 'attendances/destroy'
+  root "events#index"
+
+  resources :users, only: [:new, :create, :show]
+
+  get '/signup', to: 'users#new'
+  post '/signup', to: 'users#create'
+  get '/show', to: 'users#show'
```

Comment:
Unnecessary. So it can remove from here.

Reply:
done